Gateway/Validator/RefundResponseValidator.php

```diff
@@ -46,7 +46,7 @@ public function validate(array $validationSubject)
         $errorMessages = [];
 
         foreach ($responses as $response) {
-            if (empty($response['response']) || $response['response'] != '[refund-received]') {
+            if (empty($response['status']) || $response['status'] != 'received') {
```

Comment:
I think we can do:

```suggestion
            if ($response['status'] !== 'received') {
```

Reply:
I think if are we do different handling for each condition then it make sense to can keep both
But since we have the same handling for both conditions and we can represent them with only one condition like the suggestion, then it make the code more simple to use the suggestion